components/yaml/introduction.rst

```diff
@@ -148,6 +148,20 @@ error if something goes wrong by adding the filename to the message.
 
 .. _components-yaml-dump:
 
+Objects for Mappings
+....................
+
+.. versionadded:: 2.6
+    Support for parsing mappings as objects was introduced in Symfony 2.6.
```

Comment:
I think we should prefer the static API (see #6598) so using 2.7 here is the best idea imo.